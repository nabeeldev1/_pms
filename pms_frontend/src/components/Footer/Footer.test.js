import React from 'react';
// import ReactDOM from 'react-dom';
import Footer from '../Footer/Footer';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import classes from './Footer.css';

describe('Component: Footer', () => {
//   const items = ['Learn react', 'rest', 'go out'];

  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <Footer />
     ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});