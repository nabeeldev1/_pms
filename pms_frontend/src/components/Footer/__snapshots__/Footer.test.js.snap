// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer should match its empty snapshot 1`] = `
<footer
  className={undefined}
>
  Â© All Rights Reserved.
</footer>
`;
